Documentation/git-config.txt

```diff
@@ -10,8 +10,8 @@ SYNOPSIS
 --------
 [verse]
 'git config' [<file-option>] [--type=<type>] [--fixed-value] [--show-origin] [--show-scope] [-z|--null] <name> [<value> [<value-pattern>]]
```

Comment:
It should probably also work for `git config --comment=Yoohoo a.b c`.

Reply:
Alas, it does not work in its current state. `git config --comment=kaboom section.foo bar` causes an invocation error. However, I agree that users would _expect_ the command as it is written to work. I imagine it would require an additional `ACTION_MUMBLE` flag, but I can't seem to figure out which one?

Reply:
I did update `git_config_set_in_file_gently()`. It works, and the complete test suite runs without problems, but having to modify so many files feels inelegant.